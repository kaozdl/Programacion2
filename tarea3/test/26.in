# buscar_abb
insertar_binario (15,q) .
buscar_subarbol q .
buscar_subarbol t .
insertar_binario (12,d) .
insertar_binario (18,o) (21,v) .
insertar_binario (30,t) .
buscar_subarbol q .
buscar_subarbol v .
buscar_subarbol t .
buscar_subarbol o .
buscar_subarbol m .
Fin
