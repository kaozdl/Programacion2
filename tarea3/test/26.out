1>
2>
(15,q)
0
0
3>
(15,q)
4>
5>
(15,q)
-(12,d)
1
0
6>
-(21,v)
(15,q)
--(18,o)
-(12,d)
2
1
7>
-(21,v)
--(30,t)
(15,q)
--(18,o)
-(12,d)
2
2
8>
-(21,v)
--(30,t)
(15,q)
--(18,o)
-(12,d)
9>
(21,v)
-(30,t)
10>
(30,t)
11>
(18,o)
12>
13>Fin.
